src/main/java/HW9/FileNavigator.java

```diff
@@ -0,0 +1,62 @@
+package HW9;
+
+import java.util.*;
+
+public class FileNavigator {
+    private final Map<String, List<FileData>> records = new HashMap<>();
+    public void add(String path, String nameOfFile, long sizeOfFile){
```

Comment:
```suggestion
    public void add(String path, FileData fileData){
```
тоді зможеш зробити останній пункт